ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterAggregator.java

```diff
@@ -39,28 +41,37 @@
   }
 
   /**
-   * Take a {@link List} of JSON string representation of {@link StatsWrapper} objects and perform cluster wide
+   * Take a {@link Map} of instance name to JSON string representation of {@link StatsWrapper} objects and perform cluster wide
    * aggregation with them.
-   * @param statsWrappersJSON a {@link List} of JSON string representation of {@link StatsWrapper} objects from the
+   * @param statsWrappersJSON a {@link Map} of instance name to JSON string representation of {@link StatsWrapper} objects from the
```

Comment:
nope. This method will be called for all instances. Remember this is cluster wide aggregation task. In simple terms, task is to go over health report of every instance in the cluster and aggregate it to find the cluster wide stats.